bank/services/stoneBaas/builders.go

```diff
@@ -0,0 +1,111 @@
+package stoneBaas
+
+import (
+	"fmt"
+	"html/template"
+	"math"
+	"math/rand"
+	"strconv"
+	"strings"
+
+	"github.com/mundipagg/boleto-api/certificate"
+	"github.com/mundipagg/boleto-api/config"
+	"github.com/mundipagg/boleto-api/tmpl"
+	"github.com/mundipagg/boleto-api/util"
+)
+
+func getHeaders() map[string]string {
+	apiKey := config.Get().BaasApiKey
+
+	m := map[string]string{
+		"Content-Type": "application/json",
+		"X-API-KEY":    apiKey,
+	}
+
+	return m
+}
+
+func mapResponseHeaders(resHeaders map[string]interface{}) map[string]string {
+	m := make(map[string]string)
+
+	for k, v := range resHeaders {
+		m[k] = fmt.Sprint(v)
+	}
+
+	return m
+}
+
+func getCustomId(serviceUser string) string {
+	switch {
+	case serviceUser == "onev2":
+		return "GATEWAY"
+	case serviceUser == "pagarme":
+		return "PSP"
+	default:
+		return ""
+	}
+}
+
+func getWebhookUrl() string {
+	return config.Get().BaasWebhookUrl
+}
+
+func getValue(percentageStr string) int64 {
+	numericalPart := strings.ReplaceAll(percentageStr, ".", "")
+	value, _ := strconv.ParseInt(numericalPart, 10, 64)
+	return value
+}
+
+func buildFuncMaps() template.FuncMap {
+	funcMapsDefault := tmpl.GetFuncMaps()
+
+	return template.FuncMap{
+		"getWebhookUrl":                       getWebhookUrl,
+		"getCustomId":                         getCustomId,
+		"getValue":                            getValue,
+		"enDate":                              funcMapsDefault["enDate"],
+		"datePlusDays":                        funcMapsDefault["datePlusDays"],
+		"truncateOnly":                        funcMapsDefault["truncateOnly"],
+		"float64ToStringTruncate":             funcMapsDefault["float64ToStringTruncate"],
+		"float64ToString":                     funcMapsDefault["float64ToString"],
+		"convertAmountInCentsToPercent":       funcMapsDefault["convertAmountInCentsToPercent"],
+		"convertAmountInCentsToPercentPerDay": funcMapsDefault["convertAmountInCentsToPercentPerDay"],
+		"onlyAlphanumerics":                   funcMapsDefault["onlyAlphanumerics"],
+		"onlyOneSpace":                        funcMapsDefault["onlyOneSpace"],
+	}
+}
+
+func (b *bankStoneBaas) configureMTLS() error {
+	var err error
+
+	certificates := certificate.TLSCertificate{
+		Crt: config.Get().BaasMtlsCert,
+		Key: config.Get().BaasMtlsKey,
+	}
+
+	transportTLS, err = util.BuildTLSTransport(certificates)
+
+	b.trasportTLS = transportTLS
+
+	return err
+}
+
+func formatErrorMessage(response StoneBaasFailed) string {
+	var paramMsg string
+	messages := strings.Join(response.Messages, ", ")
+
+	if response.Param != "" {
+		paramMsg = fmt.Sprintf("O parâmetro %v está com erros: ", response.Param)
+	}
+
+	return fmt.Sprintf("%v%v", paramMsg, messages)
```

Comment:
Opa, aqui não precisa de um espaço entre os 2 %v não ? 😀 
```suggestion
	return fmt.Sprintf("%v %v", paramMsg, messages)
```